tests/integration_tests/dashboards/security/security_dataset_tests.py

```diff
@@ -37,39 +37,36 @@
 class TestDashboardDatasetSecurity(DashboardTestCase):
     @pytest.fixture
     def load_dashboard(self):
-        with app.app_context():
```

Comment:
The app context is not needed within the confines of a derived Flask-Testing class.